tests/lib/cli.js

```diff
@@ -803,6 +803,29 @@ describe("cli", () => {
         });
     });
 
+    describe("when given the exit-on-fatal-error flag", () => {
+        it("should not change exit code if no fatal errors are reported", async () => {
+            const filePath = getFixturePath("exit-on-fatal-error/no-fatal-error.js");
```

Comment:
```suggestion
            const filePath = getFixturePath("exit-on-fatal-error", "no-fatal-error.js");
```

For consistency with the other test case.